sdk/compute/mgmt/src/main/java/com/azure/resourcemanager/compute/implementation/VirtualMachineImpl.java

```diff
@@ -184,6 +191,7 @@
         super(name, innerModel, computeManager);
         this.storageManager = storageManager;
         this.networkManager = networkManager;
+        this.authorizationManager = authorizationManager;
```

Comment:
Is there any usage of the `authorizationManager`?

Reply:
To create a new VirtualMachine, after polling is complete (line 1737)

```
                inner -> new VirtualMachineImpl(inner.name(), inner, this.manager(),
                    this.storageManager, this.networkManager, this.authorizationManager));
```

Reply:
ok, I found it was used in a private resource